plugins/woocommerce/src/Internal/Admin/RemoteFreeExtensions/DefaultFreeExtensions.php

```diff
@@ -669,6 +670,221 @@ public static function get_plugin( $slug ) {
 				],
 				'is_built_by_wc' => false,
 			],
+			'tiktok-for-business:alt'               => [
+				'name'           => __( 'TikTok for WooCommerce', 'woocommerce' ),
+				'image_url'      => plugins_url( '/assets/images/onboarding/tiktok.svg', WC_PLUGIN_FILE ),
+				'description'    =>
+					__( 'Create ad campaigns and reach one billion global users with TikTok for WooCommerce.', 'woocommerce' ),
```

Comment:
Tested well! @rjchow  I'm wondering if we should add a link to "TikTok for WooCommerce" as well? 🤔 

<img src="https://user-images.githubusercontent.com/4344253/194201548-106a26c3-da9d-4d45-bec5-30ff16a6d6f3.png" width="500px" />

Reply:
fixed! nice catch